core/tests/test_contacts_crud.py

```diff
@@ -12,7 +13,7 @@ def test_add_name(setup_db):
         ContactPayload(name="Joe"),
     )
 
-    assert result.message == "Contact created."
+    assert result.message == InfoMessages.CONTACT_CREATED
```

Comment:
Forgot to mention, enam in Python has one unpleasant feature, in order to read the real value, it is necessary to get the `value` property like this `InfoMessages.CONTACT_CREATED.value`

Reply:
so basically here you compare string with object and tests fail

Reply:
Ooo. Thanks a lot 😅

Reply:
Yeah, I know, thats feels strange, like a lot of things in python